src/Operators.jl

```diff
@@ -67,15 +76,13 @@ sub(x, y) = x - y
 mult(x, y) = x * y
 pow_abs(x, y) = abs(x)^y
```

Comment:
`pow_abs` is another one I think should be converted, so that it returns NaN if `x <= 0`.

Reply:
Oh, I misunderstood what `@deprecate` did. I will change this one too.